### PR TITLE
stage/prod settings files for Oregon cluster

### PR DESCRIPTION
**Pull request by metadave**
yes, this is mostly a copy/paste of the dev settings, but we need them to get started. Prod will most likely have higher replica settings etc. I'm starting with a single `oregon` directory to keep both oregon-a and oregon-b settings, but I could see the need to split them out soonish.

### CODE REVIEW

**Comment by glogiotatidis** [comment]
Thanks Dave!

File: k8s/regions/oregon/prod.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# General settings
+TARGET_ENVIRONMENT: prod
+AWS_REGION: us-west-2
+AWS_RESOURCE_STACK: SUMO-prod
```
Comment (glogiotatidis) [inline k8s/regions/oregon/prod.yaml]: Are those two AWS_* variables used somewhere?

Comment (metadave) [reply k8s/regions/oregon/prod.yaml]: doesn't look like it, it was carried over from MDN automation.

Comment (metadave) [reply k8s/regions/oregon/prod.yaml]: removed in https://github.com/mozilla/kitsune/pull/3085

**Pull request merged**